sql/core/src/test/scala/org/apache/spark/sql/streaming/continuous/ContinuousSuite.scala

```diff
@@ -259,10 +259,10 @@ class ContinuousStressSuite extends ContinuousSuiteBase {
     testStream(df, useV2Sink = true)(
       StartStream(Trigger.Continuous(2012)),
```

Comment:
just curios: Is this value still `2012`? Can a reduction to `1012` reduce time?

Reply:
I think 2012 is safer here as it only increases to epoch 1. Smaller value might make this test flaky.

Reply:
Got it, thanks